assets/scripts/dialogs/AnalyticsDialog.jsx

```diff
@@ -40,23 +72,41 @@ function AnalyticsDialog (props) {
   const averageTotal = segmentData.map(item => item.capacity.average).reduce(sumFunc)
   const potentialTotal = segmentData.map(item => item.capacity.potential).reduce(sumFunc)
 
+  const summary = (<FormattedMessage
+    id="analytics.streetSummary"
+    defaultMessage="Your street has an estimated average traffic of {averageTotal} passengers per hour, and potential for up to {potentialTotal} passengers per hour."
+    values={{
+      averageTotal: <b>{FormatNumber(props.locale, averageTotal)}</b>,
+      potentialTotal: <b>{FormatNumber(props.locale, potentialTotal)}</b>
+    }}
+  />)
+
+  const rolledUp = rollUpCategories(segmentData)
+  const chartMax = Math.max(...rolledUp.map(item => Number.parseInt(item.capacity.potential, 10))) + 1000
+
   return (
     <Dialog>
       {(closeDialog) => (
         <div className="analytics-dialog">
           <header>
             <h1>
-              <FormattedMessage id="dialogs.analytics.heading" defaultMessage="analytics Streetmix." />
+              <FormattedMessage id="dialogs.analytics.heading" defaultMessage="Analytics" />
             </h1>
           </header>
           <div className="dialog-content">
             <div className="analytics-dialog-content">
-              <h2>Summary</h2>
               <div>
-                Your street has an estimated average traffic of <b>{FormatNumber(props.locale, averageTotal)}</b> passengers per hour, and potential for up to <b>{FormatNumber(props.locale, potentialTotal)}</b> passengers per hour.
+                {summary}
+              </div>
+              {rolledUp.map((item, index) => (item.capacity.average > 0) && <SegmentAnalytics index={index} {...item} chartMax={chartMax} />)}
+              <div className="dialog-actions">
+                <button onClick={() => saveCsv(rolledUp, props.street.name)}>
+                  <FormattedMessage id="btn.exportCsv" defaultMessage="Export as CSV" />
```

Comment:
Add this to translation file and use dash-naming convention instead of camelCase.
Because this is specific to the analytics dialog for now we can put the string inside the `dialogs.analytics` category instead of `btn`.